tests/loggers/test_mlflow_logger.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+import csv
+from pathlib import Path
+
+import pytest
+import yaml
+from torch.utils.data import DataLoader
+
+from composer.algorithms import SqueezeExcite
+from composer.loggers import MLFlowLogger
+from composer.trainer import Trainer
+from tests.common.datasets import RandomImageDataset
+from tests.common.models import SimpleConvModel
+
+
+def test_mlflow_experiment_set_up_correctly(tmp_path):
+    mlflow = pytest.importorskip('mlflow')
+
+    mlflow_uri = tmp_path / Path('my-test-mlflow-uri')
+    mlflow_exp_name = 'my-test-mlflow-exp'
+    mlflow_run_name = 'my-test-mlflow-run'
+
+    test_mlflow_logger = MLFlowLogger(experiment_name=mlflow_exp_name,
+                                      run_name=mlflow_run_name,
+                                      tracking_uri=mlflow_uri)
+
+    dataset_size = 16
```

Comment:
It doesn't look like this test actually needs a trainer? you can just call init on the logger. the other test tests its integration with the trainer and that metrics get logged.

Reply:
good call. I was just too lazy to make a mock logger and state. fixed